docs/tutorial/snapcraft-guide.md

```diff
@@ -0,0 +1,172 @@
+# Snapcraft Guide (Ubuntu Software Center & More)
+
+This guide provides information on how to package your Electron application
+for any Snapcraft environment, including the Ubuntu Software Center.
+
+## Background and Requirements
+
+Together with the broader Linux community, Canonical aims to fix many of the
+common software installation problems with the [`snapcraft`](snapcraft.io)
```

Comment:
Missing the `https://`

Reply:
(It's at the very end, this is a named link)